src/tools/w3dviewer/emittergeneralproppage.h

```diff
@@ -59,5 +61,6 @@ class EmitterGeneralPropPageClass : public CPropertyPage
     CString m_textureName;
     ShaderClass m_shader;
     float m_lifetime;
+    float m_futureStartTime;

```

Comment:
This new member is not initialized in constructor.

Reply:
fixed.